libs/api-services/src/lib/artifacts/getAllByUserId.ts

```diff
@@ -0,0 +1,11 @@
+import { artifactSummary, prisma } from '@dnd-assistant/prisma';
+
+export const getAllByUserId = async (userId: string) => {
```

Comment:
[nb] Could we perhaps call this `getArtifactsForUserId`? Hoping to keep our queries easy to identify when a file is open since you only see filename